### ISSUE TITLE
format wrong

### ISSUE THREAD

**Issue by okxiaoliang4** [johnsoncodehk/volar#1831]
`v0.40.9` format no reaction.

`v0.40.8` format wrong indent. like this
```ts
<script setup lang="ts">
onMounted(async () => {
const res = await uni.login({}) as unknown as { code: string }
apis.auth.wechatLogin({
code: res.code
}).then(res => {
console.log(res);
})
})
</script>
```

`v0.40.7` everything is ok.

**Comment by johnsoncodehk** [comment]
Can you reproduce this problem in https://github.com/johnsoncodehk/volar-starter?

**Comment by okxiaoliang4** [comment]
repo: https://github.com/okxiaoliang4/volar-starter
using takeover mode. 
file: `src/user.ts`
but `.vue` not reproduct

**Comment by johnsoncodehk** [comment]
Fixed by #1859 side effect.
